samples/nrf_profiler/sample.yaml

```diff
@@ -7,12 +7,12 @@ tests:
     integration_platforms:
       - nrf52dk_nrf52832
       - nrf52840dk_nrf52840
-      - nrf9160dk_nrf9160_ns
+      - nrf5340dk_nrf5340_cpuapp_ns
       - nrf21540dk_nrf52840
     platform_allow:
       - nrf52dk_nrf52832
       - nrf52840dk_nrf52840
-      - nrf9160dk_nrf9160_ns
```

Comment:
I would suggest to keep the `nrf9160dk_nrf9160_ns` as is and add the `nrf5340dk_nrf5340_cpuapp_ns` under `platform_allow` if needed. I think it would be good to test against nRF91 (nRF Profiler is a generic tool).